requirements-dev.txt

```
@@ -1,5 +1,5 @@
 pip~=22.0
-git+https://github.com/PennyLaneAI/pennylane.git@master
+git+https://github.com/PennyLaneAI/pennylane.git@tf-device-float32
```

Commit this after https://github.com/PennyLaneAI/pennylane/pull/5446 is merged.

```suggestion
git+https://github.com/PennyLaneAI/pennylane.git@master
```